ansible/roles/test/tasks/continuous_link_flap.yml

```diff
@@ -134,9 +134,9 @@
 
     - name: include continuous_peer_link_flap_helper.yml
       include_tasks: continuous_link_flap/continuous_peer_link_flap_helper.yml
-      # vars:
-      #   interface: "{{item}}"
-      # with_items: "{{ device_conn.keys() }}"
+      vars:
+        interface: "{{item}}"
+      with_items: "{{ device_conn.keys() }}"
```

Comment:
@praveen-li @qiluo-msft  what was the reason to comment it out in initial commit?
https://github.com/Azure/sonic-mgmt/commit/45df992d4ba67239acc606ba39e8c5b2d5c35c2f#diff-82480ca497298eac7087aac65aa1b64fR127

Reply:
It does not seem intentional to me, I see this change in my 3rd commit (#810) but do not see any comment related to it.